bookmarks.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/styles.css" />
+    <title>Quizzy</title>
+  </head>
+  <body>
+    <header>
+      <img
+        src="/pictures/quiz-comic-pop-art-style_175838-505.jpg"
+        class="logo"
+      />
+    </header>
+    <section class="card" class="cardopen">
+      <p>
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nostrum,
+        cupiditate. Cum officiis consectetur amet ab libero error necessitatibus
+        ea facilis maiores excepturi quidem in, obcaecati, culpa asperiores.
+        Voluptatum, rem dolore?
+      </p>
+      <button type="button" class="buttonhide">Show Answer</button>
+      <p class="answer">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nostrum,
+        cupiditate. Cum officiis consectetur amet ab libero error necessitatibus
+        ea facilis maiores excepturi quidem in, obcaecati, culpa asperiores.
+        Voluptatum, rem dolore?
+      </p>
+      <ul class="liste">
+        <li><button type="button">#html</button></li>
+        <li><button type="button">#flexbox</button></li>
+        <li><button type="button">#css</button></li>
+      </ul>
+      <img src="/pictures/vollebookmark.png" class="bookmarkclosed" />
+    </section>
+    <br />
+    <br />
+    <br />
+    <footer>
+      <nav>
+        <a href="/index.html"
+          ><img src="/pictures/1946488.png" class="logosfooter"
+        /></a>
+        <a
+          ><img
+            src="/pictures/kisspng-bookmark-computer-icons-like-button-instagram-face-5ada502def3d14.3322865215242568139799.png"
+            class="logosfooter"
+        /></a>
+        <a><img src="/pictures/img_322460.png" class="logosfooter" /></a>
```

Comment:
ich glaube bei denen, die noch nirgendwo hinführen, könntest du schon href="#" eingeben, damit sich der cursor so verhält, als würde er über einen Link gehen. Dann könntest du auch mit :hover in CSS noch weitere Einstellungen machen :)